src/UI/processesstorage.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <map>
+#include "processinfo.h"
+
+class IProcessesStorage{
```

Comment:
You don't need this interface because you have only single derived class and you are using it directly even in tests

Reply:
And like you said i don't use that interface i use ProcessStorage directly.